CONTRIBUTING.md

```diff
@@ -61,21 +74,35 @@ Now you can access the cluster using kubectl, e.g.: `kubectl get ns`.
 To set up a containerized Splunk instance, you can use either podman or docker.
 Follow these steps:
 
-1. Build and Run the Splunk instance by running the below cmd:
+1. Copy the splunk template file and replace the template's variables
+   with actual values. for example: change `{{ .PodName}}` to `splunk-pod`
+2. Build and Run the Splunk instance by running the below cmd:
    ```
    podman play kube splunk/splunk_container_template.yaml
    ```
     For more information about the specific command inputs and options, refer
     to the [Splunk documentation][CS1].
-2. Once the container is running, you can log in to the Splunk instance using
+3. Once the container is running, you can log in to the Splunk instance using
    the username `admin` and the password you set up.
-3. To access the Splunk [REST API][CS2],
+4. To access the Splunk [REST API][CS2],
    you can make API calls from outside the container using the `curl` command.
    For example, you can run the following command to search all data:
      ```
      curl -u admin:YourPassword -k https://localhost:8089/services/search/jobs -d search="search *"
      ```
-4. If you want to use the Splunk UI, open a web browser on the host and navigate to
+    you may also configure a .netrc file and authenticate with it instead.
```

Comment:
Since its the same file for eveyone, lets just commit it to the repo, and instruct people how to use it. In fact we probably already have this file in the repo already for use by the tests.

Reply:
indeed it is committed, fixed.